scikit-hep/pylhe
# test: Add unit tests to CI

Started to add some unit tests (including an updated CI script) and coverage check for the code.

- [x] Summarize commit messages into a comprehensive review of the PR

```
* Require scikit-hep-testdata>=0.3.1 for LHE file
* Add tests run with pytest
* Add coverage summary and checking with Codecov
   - Add associated config files: codecov.yml, pytest.ini, .coveragerc
* Add MANIFEST.in check to publishing workflow
```
